pkg/config/config.go

```diff
@@ -213,7 +213,7 @@ type ContainersConfig struct {
 	// UserNS indicates how to create a User namespace for the container
 	UserNS string `toml:"userns,omitempty"`
 
-	// UserNSSize how many UIDs to allocate for automatically created UserNS
+	// UserNSSize Deprecated, no user of this field is known.
```

Comment:
You should use this format
```
// UserNSSize how many UIDs to allocate for automatically created UserNS
// Deprecated: no user of this field is known
UserNSSize int `toml:"userns_size,omitempty,omitzero"`
```
This will make certain linters complain if it used anywhere.